src/System/Status/StatusChecker.php

```diff
@@ -518,19 +518,23 @@ public static function getFilesystemStatus($public_only = true): array
                     'status' => self::STATUS_OK
                 ]
             ];
-           // Check session dir (useful when NFS mounted))
-            if (!is_dir(GLPI_SESSION_DIR)) {
-                $status['session_dir'] = [
-                    'status' => self::STATUS_PROBLEM,
-                    'status_msg'   => sprintf(_x('glpi_status', '%s variable is not a directory'), 'GLPI_SESSION_DIR')
-                ];
-                $status['status'] = self::STATUS_PROBLEM;
-            } else if (!is_writable(GLPI_SESSION_DIR)) {
-                $status['session_dir'] = [
-                    'status' => self::STATUS_PROBLEM,
-                    'status_msg'   => sprintf(_x('glpi_status', '%s variable is not writable'), 'GLPI_SESSION_DIR')
-                ];
-                $status['status'] = self::STATUS_PROBLEM;
+            if (strtolower(ini_get('session.save_handler')) === 'files') {
```

Comment:
In this line of code, you assume that `session.save_handler` `files` value may contains uppercase chars. If this is a potential case, this `strtolower()` operation should be used in all the other checks introduced in this PR.